src/components/portrait/portrait-initials.component.js

```diff
@@ -116,7 +112,7 @@ PortraitInitials.propTypes = {
   /** Use a dark background. */
   darkBackground: PropTypes.bool,
   /** The shape of the Portrait. */
-  shape: PropTypes.oneOf([...OptionsHelper.shapesVaried, ...OptionsHelper.shapesPortrait]),
+  shape: PropTypes.oneOf(OptionsHelper.shapesPortrait),
```

Comment:
i think array is needed for correct prop checking? 

```suggestion
  shape: PropTypes.oneOf([OptionsHelper.shapesPortrait]),
```

Reply:
It is an array already